src/ml_gym/frontend/dashboard/src/App.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Template from './template/template';
+import FlowBoard from './routes/flow_board'
+import AnalysisBoard from './routes/analysis_board'
+import Settings from './routes/settings'
```

Comment:
Codacy has a fix for the issue: [Replace `'./routes/settings'` with `"./routes/settings";`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
import Settings from "./routes/settings";
```